tests/decorators/test_security.py

```diff
@@ -63,16 +64,22 @@ async def get_tokeninfo_response(*args, **kwargs):
     client.get = get_tokeninfo_response
     monkeypatch.setattr(SecurityHandlerFactory, 'client', client)
 
-    with pytest.raises(OAuthScopeProblem, match="Provided token doesn't have the required scope"):
+    with pytest.raises(OAuthScopeProblem) as exc_info:
         await wrapped_func(request)
 
+    assert exc_info.value.status_code == 403
+    assert exc_info.value.detail == "Provided token doesn't have the required scope"
```

Comment:
I believe this is because we are now subclassing connexion exceptions from starlette now, which have a different `str` representation